tutorials/sequencing.md

```
@@ -0,0 +1,31 @@
+* `SubmitRollupTransaction` relays the rollup transactions from Rollkit rollup to the sequencing network 
+* `GetNextBatch` returns the next batch of transactions along with a deterministic timestamp
+* `VerifyBatch` validates the sequenced batch
+
+### Sequencing Implementations
+
+An implementation of the sequencing interface mainly acts as a middleware that connects Rollkit rollup and the sequencing layer. It implements the sequencing interface functions described above. For example, [centralized-sequencer](https://github.com/rollkit/centralized-sequencer) is the refactored functionality from the Rollkit prior to `v0.13.8`. The centralized sequencer is the middleware run by the aggregator node of the Rollkit rollup. The aggregator node relays rollup transactions to centralized sequencer which then submits them to the DA network (such as Celestia). The header producer node then retrieves (via `GetNextBatch`) the batched transaction from the centralized sequencer to execute the transactions and produce the updated rollup state. Similarly, there are other sequencing middlewares which can be built for various sequencing strategies or even for connecting to different third-party sequencing networks. 
+
+The sequencing implementations that are currently work in progress:
+* [centralized-sequencer](https://github.com/rollkit/centralized-sequencer)
+* [based-sequencer](https://github.com/rollkit/based-sequencer)
+* [forced-inclusion-sequencer]()
```

Markdownlint

No empty links

(MD042, no-empty-links)